src/js/gi/components/profile/AdditionalInformation.jsx

```diff
@@ -125,21 +125,21 @@ export class AdditionalInformation extends React.Component {
                 <a onClick={this.props.onShowModal.bind(this, 'facilityCode')}>VA facility code:</a>
                 &nbsp;
               </strong>
-              {+it.facilityCode === 0 ? 'N/A' : +it.facilityCode}
+              {it.facilityCode || 'N/A'}
             </p>
             <p>
               <strong>
                 <a onClick={this.props.onShowModal.bind(this, 'ipedsCode')}>ED IPEDS code:</a>
                 &nbsp;
               </strong>
-              {+it.cross === 0 ? 'N/A' : +it.cross}
+              {it.cross || 'N/A'}
             </p>
             <p>
               <strong>
                 <a onClick={this.props.onShowModal.bind(this, 'opeCode')}>ED OPE code:</a>
                 &nbsp;
               </strong>
-              {+it.ope6 === 0 ? 'N/A' : +it.ope6}
+              {it.ope || 'N/A'}
```

Comment:
Yup, we were incorrectly using the 6-digit value (`ope6`) when we should've been using the full 8-digit value (`ope`).